src/lib/core/say.c

```diff
@@ -60,7 +60,7 @@ int log_level = S_INFO;
  * it's log level not less that this. -1 is used if we should not pass logs
  * to flight recorder (it is currently disabled).
  */
-static int log_level_flightrec = -1;
+int log_level_flightrec = -1;
```

Comment:
Nit: Please add a comment saying that it isn't static, because it's used in Lua via FFI.

Reply:
Done.